### PR TITLE
Make deploy optional

### PR DESCRIPTION
**Pull request by popzxc**
Add a possibility to not deploy an artifact (e.g. it's already deployed and you just want to start a new instance).

cc @vitvakatu

### CODE REVIEW

**Review by vitvakatu — APPROVED** [review]

File: exonum_launcher/configuration.py [hunk 1]
```diff
@@ -11,14 +11,18 @@ class Artifact:
...
     def from_dict(data: Dict[Any, Any]) -> "Artifact":
         """Parses an `Artifact` entity from provided dict."""
-        return Artifact(name=data["name"], runtime=data["runtime"], spec=data.get("spec", dict()))
+        spec = data.get("spec", dict())
+        # Check whether we need to deploy artifact or not
+        deploy = data.get("deploy", True)
+        return Artifact(name=data["name"], runtime=data["runtime"], spec=spec, deploy=deploy)
 
-    def __init__(self, name: str, runtime: str, spec: Any) -> None:
+    def __init__(self, name: str, runtime: str, spec: Any, deploy: bool) -> None:
```
Comment (aleksuss) [inline exonum_launcher/configuration.py:19]: What do you think about to make a default value to `deploy` parameter is `True` ? Can it make sense ?

**Review by aleksuss — APPROVED** [review]

File: exonum_launcher/configuration.py (hunk 1 above)
Comment (vitvakatu) [reply exonum_launcher/configuration.py:19]: It's is `true` by default: 
```python
# Check whether we need to deploy artifact or not
deploy = data.get("deploy", True)
```

Comment (popzxc) [reply exonum_launcher/configuration.py:19]: For parsing a config it already is: https://github.com/popzxc/exonum-launcher/pull/22/files#diff-e56cda788227a2cee14187042fbb4d96R16

If somebody will decide to create an `Artifact` entity manually, I think it's better to keep things explicit.

Comment (aleksuss) [reply exonum_launcher/configuration.py:19]: Haven't seen it. Cool ! 👍🏻

**Pull request merged**